resources/testdriver.js

```diff
@@ -49,6 +49,54 @@
      * @namespace {test_driver}
      */
     window.test_driver = {
+        /**
+         Represents `WebDriver BiDi <https://w3c.github.io/webdriver-bidi>`_ protocol.
+         */
+        bidi: {
+            /**
+             * `log <https://w3c.github.io/webdriver-bidi/#module-log>`_ module.
+             */
+            log: {
+                /**
+                 * `log.entryAdded <https://w3c.github.io/webdriver-bidi/#event-log-entryAdded>`_ event.
+                 */
+                entry_added: {
+                    /**
+                     * Subscribe to the `log.entryAdded` event. This does not add actual listeners. To listen to the
+                     * event, use `on` method.
+                     * @param {{contexts?: null | (string | Window)[]}} props - Parameters for the subscription.
+                     * * `contexts`: an array of window proxies or browsing context ids to listen to the event. If not
+                     *   provided, the event subscription is done for the current window's browsing context. `null` for
+                     *   the global subscription.
+                     * @return {Promise<void>}
+                     */
+                    subscribe: async function (props = {}) {
+                        return window.test_driver_internal.bidi.log.entry_added.subscribe(props);
```

Comment:
In other places where a promise is returned, the `test_driver` function just returns the result of the `test_driver_internal` function, without `async` or `await`. Aligning with that and also going with `params` as the argument name:

```suggestion
                    subscribe: function (params = {}) {
                        return window.test_driver_internal.bidi.log.entry_added.subscribe(params);
```